boards/heltec_vision_master_e213.json

```diff
@@ -0,0 +1,39 @@
+{
+  "build": {
+    "arduino": {
+      "ldscript": "esp32s3_out.ld",
+      "partitions": "default_8MB.csv"
+    },
+    "core": "esp32",
+    "extra_flags": [
+      "-DBOARD_HAS_PSRAM",
+      "-DHELTEC_VISION_MASTER_E213",
```

Comment:
I think maybe(?) we should just remove `"-DHELTEC_VISION_MASTER_E213"` from here, just because it looks like the other boards.json files leave this definition to variant.h

Reply:
removed